packages/app-store/routing-forms/pages/form-edit/[...appPages].tsx

```diff
@@ -50,431 +38,40 @@ const appendArray = <T,>({
-                  check: () => hookFormFields.length > 1,
-                  fn: () => {
-                    removeHookFormField(key);
-                  },
-                }}
-                moveUp={{
-                  check: () => key !== 0,
-                  fn: () => {
-                    swapHookFormField(key, key - 1);
-                  },
-                }}
-                moveDown={{
-                  check: () => key !== hookFormFields.length - 1,
-                  fn: () => {
-                    if (key === hookFormFields.length - 1) {
-                      return;
-                    }
-                    swapHookFormField(key, key + 1);
-                  },
-                }}
-                key={key}
-              />
-            );
-          })}
-        </div>
-        {hookFormFields.length ? (
-          <div className={classNames("flex")}>
-            <Button
-              data-testid="add-field"
-              type="button"
-              StartIcon="plus"
-              color="secondary"
-              onClick={addField}>
-              Add field
-            </Button>
-          </div>
-        ) : null}
-      </div>
-    </div>
-  ) : (
-    <div className="bg-default w-full">
-      <EmptyScreen
-        Icon="file-text"
-        headline="Create your first field"
-        description="Fields are the form fields that the booker would see."
-        buttonRaw={
-          <Button data-testid="add-field" onClick={addField}>
-            Create Field
-          </Button>
-        }
-      />
-    </div>
-  );
-};
-
 export default function FormEditPage({
   appUrl,
   ...props
 }: inferSSRProps<typeof getServerSideProps> & { appUrl: string }) {
+  const { t } = useLocale();
+  const formMethods = useFormContext<FormValues>();
+
   return (
     <SingleForm
       {...props}
       appUrl={appUrl}
-      Page={({ hookForm, form }) => <FormEdit appUrl={appUrl} hookForm={hookForm} form={form} />}
+      Page={({ hookForm, form }) => (
+        <>
+          <FormBuilder
+            title={t("booking_questions_title")}
+            description={t("booking_questions_description")}
+            addFieldLabel={t("add_a_booking_question")}
+            formProp="fields"
+            dataStore={{
+              options: {
+                locations: {
+                  // FormBuilder doesn't handle plural for non-english languages. So, use english(Location) only. This is similar to 'Workflow'
+                  source: { label: "Location" },
+                  value: getLocationsOptionsForSelect(formMethods.getValues("locations") ?? [], t),
+                },
+              },
+            }}
+            shouldConsiderRequired={(field: BookingField) => {
+              // Location field has a default value at backend so API can send no location but we don't allow it in UI and thus we want to show it as required to user
+              return field.name === "location" ? true : field.required;
```

Comment:
There is no location field in Routing Form. So, let's remove this for now from here.